pya/aserver.py

```diff
@@ -121,6 +122,10 @@ def max_in_chn(self):
     def device(self):
         return self._device
 
+    @property
+    def is_active(self) -> bool:
+        return self._is_active
+
```

Comment:
Instead of using a dedicated private value here, one could derive the value from the stream:

```python
@property
def is_active(self) -> bool:
    return self.stream.is_active() if self.stream is not None else False
```

one parameter less to maintain.